### PR TITLE
Skip first line and edit street name. Fixes #1

### LINKED ISSUE #1: Erroneously prints out first line
My favourite street is Richmond. Please fix.

### PR DESCRIPTION
**Pull request by kspaceman**

### CODE REVIEW

**Comment by kspaceman** [comment]
I think this fixed the problem, but it's not tested.

File: favourite-street.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
     csv_reader = csv.reader(csv_file, delimiter=',')
     line_count = 0
     for row in csv_reader:
+        if linecount > 0:
```
Comment (isolovey) [inline favourite-street.py:10]: there's an underscore missing.

Comment (kspaceman) [reply favourite-street.py:10]: Oops.

**Comment by isolovey** [comment]
Looks good. Accepting.

**Pull request merged**